src/components/Number404/Number4.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import styles from "./style.css";
+import classnames from "classnames";
+
+interface Props {
+    side: "left" | "right";
+}
+
+const Number4: React.FC<Props> = ({ side }) => (
```

Comment:
yeah, not sure if making separate component for `4` 
writing this thing as part of `Error404Text' isn't that bad idea